.github/workflows/main.yml

```diff
@@ -40,10 +40,17 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v1
+      - name: Setup Go 1.16
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.16
+      - name: go build deps
+        run: make cmd/cli/chart.tgz pkg/envoy/lds/stats.wasm
```

Comment:
Yeah, I was struggling with this for a couple of hours. The linter is not even starting unless these files are there.

The alternative I considered was to create a dummy script that puts some empty files there just to make the linter happy. But this seemed a bit too hacky.

Reply:
Hm, that's weird. It might be worth making an empty file at least for the wasm since building that requires pulling down a pretty big docker image. That's not making the CI run as a whole any slower though. I'd say this is fine as-is and we can tweak it later if we want.

Reply:
I made a target that generates empty files just for testing purposes in the linter